src/cleartext.js

```diff
@@ -155,14 +155,17 @@ export class CleartextMessage {
 
 
 /**
- * reads an OpenPGP cleartext signed message and returns a CleartextMessage object
- * @param {String | ReadableStream<String>} armoredText text to be parsed
+ * Reads an OpenPGP cleartext signed message and returns a CleartextMessage object
+ * @param {String | ReadableStream<String>} cleartextMessage text to be parsed
```

Comment:
Can this be a stream? If so the TS definition should be updated, same for `readKeys`.

Reply:
Technically it can be, by virtue of the underlying `unarmor` function accepting a stream. But I don't think there's really any reason to do this, as it doesn't add anything (it reads the entire stream immediately).

Reply:
I think it's confusing to have mismatching docs and TS though

Reply:
Yeah, true. Anyway, it's not really the concern of this PR, we can make it consistent later 😌